webapp/sources/rudder/rudder-core/src/main/scala/com/normation/rudder/services/nodes/NodeInfoService.scala

```diff
@@ -375,7 +375,7 @@ trait NodeInfoServiceCached extends NodeInfoService with NamedZioLogger with Cac
         TimingDebugLogger.debug(s"Getting node info entries: ${t1-t0}ms")
 
         for {
-          res <- ZIO.foreach(nodes) { case (id, nodeEntry) => mapOneNode(id, nodeEntry, nodeInventories, machineInventories) }
+          res <- ZIO.foreachPar(nodes) { case (id, nodeEntry) => mapOneNode(id, nodeEntry, nodeInventories, machineInventories) }
```

Comment:
this is a great improvement

Reply:
this works too well and prevent policy generation to go fast :/

Reply:
you likely need to limite the parallelism with `foreachParN(maxParallism)(nodes) { .... }`

Reply:
yes but i don't know which value to set